src/global.h

```diff
@@ -319,11 +319,41 @@ typedef unsigned char      uint8_t;
 // definition for custom event
 #define MS_PACKET_RECEIVED 0
 
-/* Classes ********************************************************************/
+/* Exception Classes **********************************************************/
+
+class CInfoExit
```

Comment:
Although not related - maybe more to the error message redesign, but wouldn't it be better to have a class
CException
with sub-classes CInfoExit, CGenErr etc?

Reply:
@ann0see 

> "wouldn't it be better to have a class CException"

No, unfortunalely that's not the normal way exceptions are handled, but we already have the CGenErr class for Jamulus specific errors. (And CException often already exists as a standard class in libraries, i.e. in MFC)

Especially InfoExit should not be be a generic exception, since it's not an error and should be used with care (for now only in main/commandline parameter handling..)

Also ErrorExit should not be a generic exception, because we want that one always to propagate to main. So if catched anywhere else it should always be re-thrown. (So if it was a CGenErr we would have to adapt all code where a CGenErr is catched now.)

Proper exception handling in Jamulus would be a completely separate issue anyway since most exceptions are not handled at all in Jamulus, and that could be a big challenge since there probably will be different exceptions for the different OS's.

Reply:
> could be a big challenge since there probably will be different exceptions for the different OS's.

Fair point. So "not part of this PR" then.